recipes/mlxtend/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "mlxtend" %}
+{% set version = "0.7.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8c9206449d3e77ff52a288cd2741c10361bc39924b6101a777bda475b05e7f16" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.10.4
+    - scipy >=0.17
+    - scikit-learn >=0.18
+    - pandas >=0.17.1
+    - matplotlib >=1.5.1
+  run:
+    - python
+    - numpy >=1.10.4
+    - scipy >=0.17
+    - scikit-learn >=0.18
+    - pandas >=0.17.1
+    - matplotlib >=1.5.1
+
+test:
+  imports:
+    - mlxtend
+    - mlxtend._base
+    - mlxtend.classifier
+    - mlxtend.cluster
+    - mlxtend.data
+    - mlxtend.evaluate
+    - mlxtend.externals
+    - mlxtend.feature_extraction
+    - mlxtend.feature_selection
+    - mlxtend.file_io
+    - mlxtend.frequent_patterns
+    - mlxtend.math
+    - mlxtend.plotting
+    - mlxtend.preprocessing
+    - mlxtend.regressor
+    - mlxtend.text
+    - mlxtend.utils
+
+about:
+  home: https://github.com/rasbt/mlxtend
+  license: BSD v3
+  license_family: BSD
+  license_file: LICENSE-BSD3.txt
+  summary: Machine Learning Library Extensions
+  description: '
```

Comment:
There are a bunch of extra blank lines in the `description`.  You can use either a `|` or a `>` at the start for literal or folded style respectively:

http://symfony.com/doc/current/components/yaml/yaml_format.html